xilem/examples/slider_demo.rs

```diff
@@ -117,9 +117,9 @@ fn app_logic(state: &mut AppState) -> impl WidgetView<AppState> + use<> {
                 })
                 .step(5.0)
                 .disabled(!state.use_transparency)
-                .active_track_color(Color::from_rgb8(0x78, 0x71, 0x6c))
-                .thumb_color(Color::WHITE)
-                .thumb_radius(10.0),
+                .prop(BarColor(Color::from_rgb8(0x78, 0x71, 0x6c)))
+                .prop(ThumbColor(Color::WHITE))
+                .prop(ThumbRadius(10.0)),
```

Comment:
As we currently don't use `.prop` anywhere else in the examples, I wonder whether this should be abstracted in `Style`. Though either is fine for me.

Reply:
OTOH, I think the `.prop` method should have a demo in example 😄. Anyway, I don't like the currently available customization options very much. I think the whole thumb should be customizable, shape and all. As such I am against adding these style abstractions to `Style`.